scripts/rewrite-shrinkwrap.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+const fs = require("fs");
+const getPackageInfo = require(__dirname + "/utils").getPackageInfo;
+
+const _path = __dirname + "/../temp/package/npm-shrinkwrap.json";
+const data = require(_path);
+
+(async () => {
+  const filterPkgs = async (obj, key) => {
+    const _obj = {};
+    for (const pkg of Object.keys(obj[key])) {
+      if (pkg.startsWith("__tests__")) continue;
+      // if (pkg === "") continue;
+      // if (pkg === "node_modules/@zowe/cli") continue;
+      if (obj[key][pkg].dev) continue;
```

Comment:
Shouldn't testing/dev dependencies be already removed at this point? Curious about the reason for these if statements 😛

Reply:
it's possible for the SW file to still contain test and dev dependencies right before things get published to public npm.
And example is `node_modules/@babel/code-frame` contained in https://zowe.jfrog.io/zowe/api/npm/npm-local-release/@zowe/cli/-/@zowe/cli-6.37.5.tgz

the `__tests__` was more of a precaution for packages contained in https://raw.githubusercontent.com/zowe/zowe-cli/next/npm-shrinkwrap.json